core/store/store.go

```diff
@@ -101,7 +101,7 @@ func (wrapper *lazyRPCWrapper) Call(result interface{}, method string, args ...i
 	return wrapper.client.Call(result, method, args...)
 }
 
-func (wrapper *lazyRPCWrapper) Subscribe(ctx context.Context, channel interface{}, args ...interface{}) (eth.Subscription, error) {
+func (wrapper *lazyRPCWrapper) Subscribe(_ context.Context, channel interface{}, args ...interface{}) (eth.Subscription, error) {
```

Comment:
We probably shouldn't actually be throwing away the context... Can we use it instead?

Reply:
the ctx parameter was overwritten by a new context in function before it first be used, so the staticcheck warning this. For avoiding this warning, is it a better idea that leave the ctx parameter and use a newCtx instead?

Reply:
use ctx in parameters instead of create new one in function.